server/autotest_server.py

```diff
@@ -354,18 +358,21 @@ def wrapper(*args, **kwargs):
 
 ### RUN TESTS ###
 
-def copy_test_script_files(markus_address, assignment_id, tests_path, exclude):
+def copy_test_script_files(markus_address, assignment_id, tests_path):
```

Comment:
Should all \*test_script_file\* names be changed to \*test_file\* now that the scripts have lost their meaning?